src/main/services/dataCollectors/MergeConflictCollector.groovy

```diff
@@ -7,6 +7,11 @@ import util.MergeHelper
 
 import static app.MiningFramework.arguments
 
+/**
+ * @produces: a [outputPath]/mergeconflicts/results.csv file with the following format:
```

Comment:
requires git? @rafaelmotaalves 

por que usasses produces ao invés de provides? é padrao javadoc ou algo assim? se for deixa, senao troca. provides é terminologia de arquitetura de software.

Reply:
confundi o nome mesmo, vou trocar

Reply:
eu relação ao GIT não botei porque todo o framework requer o GIT, inclusive a implementação do framework em si, sem os services, então achei que não seria pertitnente, porque todos os services iriam ter